packages/core/src/App/Containers/NotificationsDialog/notifications-dialog.jsx

```diff
@@ -5,147 +5,154 @@ import { CSSTransition } from 'react-transition-group';
 import { Button, DesktopWrapper, Icon, MobileDialog, MobileWrapper, ThemedScrollbars, Text } from '@deriv/components';
 import { BinaryLink } from 'App/Components/Routes';
 import { connect } from 'Stores/connect';
-import { localize } from '@deriv/translations';
+import { localize, Localize } from '@deriv/translations';
 import { toTitleCase, isEmptyObject, isMobile } from '@deriv/shared';
 
 import { EmptyNotification } from 'App/Components/Elements/Notifications/empty-notification.jsx';
 
-class NotificationsDialog extends React.Component {
-    setWrapperRef = node => {
-        this.wrapper_ref = node;
-    };
+const NotificationsListEl = ({ notifications, toggleDialog }) => (
```

Comment:
(suggestion) renaming this to `NotificationsList` since this is not an element.

Reply:
@reza-fs ,
Thanks for comment 
Agree with you I did not change these names because the old element name is notifications_list_el. 
I will update names